mpa/modules/models/segmentors/mean_teacher_segmentor.py

```diff
@@ -0,0 +1,102 @@
+from mmseg.models import SEGMENTORS, build_segmentor
+from mmseg.models.segmentors.base import BaseSegmentor
+from mmseg.ops import resize
+import torch
+import functools
+from collections import OrderedDict
+from mpa.utils.logger import get_logger
+logger = get_logger()
+
+
+@SEGMENTORS.register_module()
+class MeanTeacherNaive(BaseSegmentor):
```

Comment:
Could you update name from `MeanTeacherNaive` -> `MeanTeacher`? `Naive` does not have ... meaningful somethings :)